nUpdate/Updating/UpdateRequirement.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using Microsoft.Win32;
+
+namespace nUpdate.Updating
+{
+    [Serializable]
+    public class UpdateRequirement
+    {
+        public enum RequirementType
```

Comment:
Please keep the guidelines in mind: "DotNetFramework" and "Registry" instead of lower case.